example-app/SantokuApp/src/bases/ui/common/types.ts

```diff
@@ -0,0 +1,4 @@
+import {restyleTheme} from '../theme/restyleTheme';
+
+export type StyledSpacingKeys = keyof typeof restyleTheme.spacing;
+export type StyledSpaceProps = {space: keyof typeof restyleTheme.spacing};
```

Comment:
```suggestion
export type StyledSpaceProps = {space: StyledSpacingKeys};
```

Reply:
Currently, Primitive parts are placed in common.
Perhaps it is not only the parts added this time, but also the existing parts.

How about we leave it as it is this time and consider it separately?

Reply:
After discussion, it was decided on a policy of placing the directory by role.

Columns and Rows will be placed in box directories.

The directory arrangement will be updated in another PR.